frontend/src/helpers/render/allocation.js

```diff
@@ -43,5 +43,10 @@ export function renderClientStatus(allocation) {
         icon = clientStatusIcon[allocation.ClientStatus];
     }
 
-    return <span className={ textColor }>{icon} {allocation.ClientStatus}</span>;
```

Comment:
The client status name was just taking up space, it can be expressed by a 10x10 icon equally good, with tooltip to quickly show the status :)